src/components/InlineCode.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropsWithChildren } from 'react';
+import tw from 'twin.macro';
```

Comment:
Oh dear, okay, this is why, we may need to finally bite the bullet and port everything over to a more modern styling system.